src/components/InventoryGroupDetail/InventoryGroupDetail.js

```diff
@@ -51,11 +76,57 @@ const InventoryGroupDetail = ({ groupId }) => {
     );
   }, [data]);
 
-  const [activeTabKey, setActiveTabKey] = useState(0);
-
   // TODO: append search parameter to identify the active tab
 
-  return (
+  const [hasEdgeImages, setHasEdgeImages] = useState(false);
+  const EdgeParityEnabled = useFeatureFlag('edgeParity.inventory-list');
+  useEffect(() => {
```

Comment:
Can we reuse the existing hook in HybridInventoryTabs with a bit of modification? I feel that we are increasing the duplicate code more and more.